package.json

```diff
@@ -172,5 +172,20 @@
   },
   "ember-addon": {
     "after": "ember-cli-legacy-blueprints"
+  },
+  "resolutions": {
+    "@glimmer/compiler": "0.38.5-alpha.1.6",
+    "@glimmer/encoder": "0.38.5-alpha.1.6",
+    "@glimmer/interfaces": "0.38.5-alpha.1.6",
+    "@glimmer/low-level": "0.38.5-alpha.1.6",
+    "@glimmer/node": "0.38.5-alpha.1.6",
+    "@glimmer/opcode-compiler": "0.38.5-alpha.1.6",
+    "@glimmer/program": "0.38.5-alpha.1.6",
+    "@glimmer/reference": "0.38.5-alpha.1.6",
+    "@glimmer/runtime": "0.38.5-alpha.1.6",
+    "@glimmer/syntax": "0.38.5-alpha.1.6",
+    "@glimmer/util": "0.38.5-alpha.1.6",
+    "@glimmer/vm": "0.38.5-alpha.1.6",
+    "@glimmer/wire-format": "0.38.5-alpha.1.6"
```

Comment:
Without these, yarn seems to be pretty liberal when interpreting the "^0.38.5-alpha.1.6" in these package's dependencies list. Should I commit this or not?

Reply:
Ya, seems fine to do here, but we could also publish a `0.38.0-alpha.1.6.1` that depends on exact versions (instead of `^`) which probably would also solve the issue.

These resolutions seem easier to me though...

Reply:
`0.38.0-alpha.1.6.1` 😭